tests/Client/UpdaterTest.php

```diff
@@ -998,15 +998,12 @@ public function providerExceptionForInvalidMetadata(): array
                     'targets' => 2,
                 ],
             ],
-            // For targets.json files, adding a new key or changing the existing version number
-            // will result in a SignatureThresholdException because currently the test
-            // fixtures do not contain hashes for targets.json files in snapshot.json.
             // § 5.6.3
             'add key to targets.json' => [
                 'targets.json',
                 ['signed', 'newvalue'],
                 'value',
-                new SignatureThresholdException("Signature threshold not met on targets"),
+                new MetadataException("The 'targets' contents does not match hash 'sha256' specified in the 'snapshot' metadata."),
```

Comment:
Does mean we lose test coverage for `Signature threshold not met on targets`? couldn't still happen especially since `hashes` is optional. 

I search the code base for "Signature threshold not met on " and I only see 2 other roles that are hard-coded

Reply:
I reverted to the status of `main` and made better-scoped changes; this line is no longer affected by this PR.